tests/phpunit/migration/LogMigrator_Test.php

```diff
@@ -7,27 +7,27 @@
  * @group migration
  */
 class LogMigrator_Test extends ActionScheduler_UnitTestCase {
-	function setUp() {
+	public function setUp() {
 		parent::setUp();
-		if ( ! taxonomy_exists( ActionScheduler_wpPostStore::GROUP_TAXONOMY )  ) {
-			// register the post type and taxonomy necessary for the store to work
+		if ( ! taxonomy_exists( ActionScheduler_wpPostStore::GROUP_TAXONOMY ) ) {
+			// register the post type and taxonomy necessary for the store to work.
 			$store = new ActionScheduler_wpPostStore();
 			$store->init();
 		}
 	}
 
 	public function test_migrate_from_wpComment_to_db() {
-		$source = new ActionScheduler_wpCommentLogger();
-		$destination = new ActionScheduler_DBLogger();
-		$migrator = new LogMigrator( $source, $destination );
-		$source_action_id = rand( 10, 10000 );
-		$destination_action_id = rand( 10, 10000 );
+		$source                = new ActionScheduler_wpCommentLogger();
+		$destination           = new ActionScheduler_DBLogger();
+		$migrator              = new LogMigrator( $source, $destination );
+		$source_action_id      = wp_rand( 10, 10000 );
+		$destination_action_id = wp_rand( 10, 10000 );
 
-		$logs = [];
-		for ( $i = 0 ; $i < 3 ; $i++ ) {
-			for ( $j = 0 ; $j < 5 ; $j++ ) {
-				$logs[ $i ][ $j ] = md5(rand());
-				if ( $i == 1 ) {
+		$logs = array();
+		for ( $i = 0; $i < 3; $i++ ) {
+			for ( $j = 0; $j < 5; $j++ ) {
+				$logs[ $i ][ $j ] = md5( wp_rand() );
+				if ( 1 === absint( $i ) ) {
```

Comment:
`absint()` seems unnecessary, we could just have a strict comparison (we can see from line 27 it will be an integer belonging to a pre-defined range).

```suggestion
				if ( 1 === $i ) {
```